components/dictionary.txt

```diff
@@ -184,9 +184,12 @@ lv
 LV
 LVs
 lysis
+MacDown
+MacOS
 macrophage
 Malhotra
 McInnes
+md
```

Comment:
Realized that you don't actually use `md` anymore!
```suggestion
```

Reply:
Womp, it was still there. Gone now!